contrib/oadp-recovery/test/create-test-clusters.sh

```
@@ -0,0 +1,202 @@
+NUM_CLUSTERS=${NUM_CLUSTERS:-2}
+NAMESPACE="test-oadp-recovery"
+
+echo "🚀 Creating $NUM_CLUSTERS test clusters directly"
+
+# Create namespace
+kubectl create namespace "$NAMESPACE" --dry-run=client -o yaml | kubectl apply -f -
+
+# Function to create a single cluster
+create_cluster() {
+    local cluster_num=$1
+    local cluster_name="test-cluster-$(printf "%02d" $cluster_num)"
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)